estimate-web/src/sagas/updateWorkItem.js

```diff
@@ -51,9 +58,26 @@ function* watchRemoveStoryPoints() {
     }
 }
 
+function* watchAddComment() {
+    while (true) {
+        const { workItemId, comment } = yield take(REQUEST_WORKITEM_ADD_COMMENT);
```

Comment:
Why do use `while + take` here but `takeEvery` in `watchGetComments`?

Reply:
No special reason. Changed to `takeEvery` for consistency